curp/src/server/curp_node.rs

```diff
@@ -998,4 +1034,22 @@ mod tests {
         assert!(curp.is_leader());
         curp.shutdown_trigger().self_shutdown_and_wait().await;
     }
+
+    fn init_curp_node(
```

Comment:
This function is never used. Do we really need it?

Reply:
Removed